Maddori.Apple/Maddori.Apple/Global/UIComponent/FeedbackTypeButtonView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  FeedbackTypeButtonView.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/10/19.
+//
+
+import UIKit
+
+import SnapKit
+
+final class FeedbackTypeButtonView: UIButton {
+    private enum FeedbackType {
+        case continueType
+        case stopType
+    }
+    private enum Size {
+        static let buttonPadding: CGFloat = 11
+        static let width: CGFloat = 158
+        static let height: CGFloat = 46
```

Comment:
p3;

```suggestion
        static let width: CGFloat = 158
        static let height: CGFloat = 46
```

buttonPadding은 사용되지 않는 것 같은데, 지워도 괜찮지 않을까요오??

Reply:
[68e685d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/27/commits/68e685de40d10b3a26a9ce71d4f906e12b8adc8c)
수정해서 반영해두었습니다 !